source/unified-test-format/tests/invalid/clientEncryptionOpts-kmsProviders-additionalProperties.yml

```diff
@@ -1,18 +0,0 @@
-description: clientEncryptionOpts-kmsProviders-additionalProperties
```

Comment:
If you're able to use `patternProperties` instead of `additionalProperties: true`, this test should remain in place and you can adjust/clone any existing tests for type-specific validation accordingly.